static/product-designer-portfolio/index.html

```diff
@@ -0,0 +1,285 @@
+            alt="description"
+            class="mx-auto max-h-[400px] lg:max-h-[400px]"
+          />
+          <img
+            src="https://images.unsplash.com/photo-1633409361618-c73427e4e206?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+            alt="description"
+            class="mx-auto max-h-[400px] lg:max-h-[400px]"
+          />
+          <img
+            src="https://images.unsplash.com/photo-1633409361618-c73427e4e206?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+            alt="description"
+            class="mx-auto max-h-[400px] lg:max-h-[400px]"
+          />
+        </div>
+      </div>
+    </section>
+    <!-- ==== End Visual Explorations ====  -->
+
+    <!-- ==== Start Personal Projects ==== -->
+    <section class="my-16">
+      <div class="container">
+        <h1 class="text-light-white mb-8 text-center text-3xl">
+          Personal Projects
+        </h1>
+        <!-- gallery -->
+        <div class="grid gap-6 lg:grid-cols-3">
+          <img
+            src="https://images.unsplash.com/photo-1612296727051-4b2fa6f2bd47?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTh8fGZpZ21hfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60"
+            alt="description"
+            class="mx-auto max-h-[600px] object-cover lg:max-h-full"
+          />
+          <img
+            src="https://images.unsplash.com/photo-1612296727051-4b2fa6f2bd47?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTh8fGZpZ21hfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60"
+            alt="description"
+            class="mx-auto max-h-[600px] object-cover lg:max-h-full"
+          />
+          <img
+            src="https://images.unsplash.com/photo-1612296727051-4b2fa6f2bd47?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTh8fGZpZ21hfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60"
+            alt="description"
+            class="mx-auto max-h-[600px] object-cover lg:max-h-full"
+          />
+        </div>
+      </div>
+    </section>
+    <!-- ==== End Personal Projects ==== -->
+
+    <!-- ==== Start Contact ==== -->
+    <section id="contact" class="mb-16 mt-20">
+      <div class="container">
+        <h1 class="text-light-white mb-8 text-center text-4xl">Contact Me</h1>
+        <p class="text-light-gray mx-auto mb-8 w-1/2 text-center text-2xl">
+          If you are looking to hire a product designer, I’m currently available
+          for freelance work
+        </p>
+        <a
+          href="mailto:your@email.com"
+          class="bg-btn mx-auto block w-fit px-8 py-4 text-white duration-300 hover:shadow-xl hover:shadow-white/20"
+        >
+          <i class="bi bi-envelope"></i>
+          your@email.com
+        </a>
+      </div>
+    </section>
+    <!-- ==== End Contact ==== -->
+
+    <!-- === Start footer ==== -->
+    <footer class="border-t-[1px] border-white py-8">
```

Comment:
I believe `border-t` itself would provide 1px of top-border width. So you can remove the arbitrary value here.